joomla/joomla-cms
# [html5] Beez3 frontend template

#### Summary of Changes

This is a redo of https://github.com/joomla/joomla-cms/pull/9842 for **beez3** only.

This PR does a basic conversion of **beez3** template to HTML5.
Also made some minor corrections:
- Use the API in inline css/js (except error page)
- Use ie9 or lower hml5.js in all files.
#### Testing Instructions
1. Install patch
2. Set the site template as "beez3" (if not yet)
3. Go to frontend and check if the the following pages are working properly:
   - normal page (test any existent URL)
   - component popup (add `?tmpl=component&print=1` to an existent page)
   - error pages (test one non existent URL)

Check also the code changes.
#### Observations

All pages should now start with this code:

``` html
<!DOCTYPE html>
<html lang="en-gb" dir="ltr">
<head>
[...]
    <meta charset="utf-8" />
[...]
```
